reddit_liveupdate/controllers.py

```diff
@@ -1090,3 +1092,38 @@ def GET_mediaembed(self, liveupdate, embed_index):
         }
 
         return pages.LiveUpdateMediaEmbedBody(**args).render()
+
+
+@add_controller
+class LiveUpdateAdminController(RedditController):
```

Comment:
I don't feel strongly, but is there any reason for this to have a separate controller and the admin page trappings? I don't see it being injected into the admin menu anywhere so it's just sorta faking being part of the admin UI right?